pkg/vectorize/add/add.go

```diff
@@ -31,6 +40,28 @@ func unsignedOverflow[T constraints.Unsigned](a, b, c T) bool {
 }
 
 func NumericAddSigned[T constraints.Signed](xs, ys, rs *vector.Vector) error {
+	return cNumericAddSigned[T](xs, ys, rs)
+}
+
+func cNumericAddSigned[T constraints.Signed](xs, ys, rs *vector.Vector) error {
+	xt, yt, rt := vector.MustTCols[T](xs), vector.MustTCols[T](ys), vector.MustTCols[T](rs)
+	flag := 0
```

Comment:
Add constants for 0,1,2 like 0 -- VECTOR_VECTOR, 1 -- SCALAR_VECTOR, 2 -- VECTOR_SCALAR.